semantic-scope-graph/src/Data/Name.hs

```diff
@@ -0,0 +1,63 @@
+{-# LANGUAGE FlexibleContexts #-}
+{-# LANGUAGE OverloadedStrings #-}
+module Data.Name
+( Name
+-- * Constructors
+, gensym
+, name
+, nameI
+, formatName
+, __self
+) where
```

Comment:
IMO we should aim to replace this module with `Analysis.Name`.

Reply:
Filed as https://github.com/github/semantic/issues/421.

Reply:
Thank you! ❤️